src/Kernels/common.hpp

```diff
@@ -126,7 +126,7 @@ constexpr unsigned int
  * @return aligned number of reals.
  **/
 constexpr unsigned int getNumberOfAlignedReals(unsigned int i_numberOfReals,
-                                               unsigned int i_alignment = ALIGNMENT) {
+                                               unsigned int i_alignment = VECTORSIZE) {
```

Comment:
Should we then also change the variable name to `i_vectorsize`?

Reply:
IMO no, because it is an alignment (to vectorsize)

Reply:
(same below)

Reply:
Ok, I've left it that way for now—since even though it's `VECTORSIZE`, we're aligning everything. Instead, we may have to rename the `ALIGNMENT` variable soon, I'd guess. And also, make both of them a C++ constexpr maybe.

I've also cleaned up the file a bit (i.e. remove `i_` etc.) and templated the functions with the size of the reals used. ... That could also become a parameter instead.